Doppler.Currency/appsettings.json

```diff
@@ -18,10 +18,10 @@
   "BnaService": {
     "Endpoint": "https://bna.com.ar/Cotizador/HistoricoPrincipales?id=billetes&filtroDolar=1&filtroEuro=0",
     "ValidationHtml": "Dolar U.S.A",
-    "NoQuotation": "There are no pending quotes for that date."
+    "NoCurrency": "There are no pending quotes for that date."
   },
   "SlackHook": {
     "Url": "[SECRET KEY]",
-    "Text": "UsdQuotation Service can't get the USD quotation of the Bank of Nacion Argentina today, Please check Html format"
+    "Text": "Doppler Currency Service can't get the USD quotation of the Bank of Nacion Argentina today, Please check Html format"
```

Comment:
These changes are not related with publish, what do you think about moving them to another PR?

Reply:
Yes, of course, I'll create another PR with more improvement for this repo